docs/BreakingChanges/024 EncoderParameter ctor is obsolete.md

```diff
@@ -0,0 +1,22 @@
+## 24: EncoderParameter ctor is obsolete
```

Comment:
{
  "Id": "24",
  "Title": "EncoderParameter ctor is obsolete",
  "Details": "The `EncoderParameter.EncoderParameter(Encoder, Int32, Int32, Int32)` constructor is obsolete now and will introduce build warnings if used.",
  "Suggestion": "Although the `EncoderParameter.EncoderParameter(Encoder, Int32, Int32, Int32)`  constructor will continue to work, the following constructor should be used instead to avoid the obsolete build warning when re-compiling code with .NET 4.5 tools: [EncoderParameter.EncoderParameter(Encoder, Int32, EncoderParameterValueType, IntPtr)](https://msdn.microsoft.com/en-us/library/hh875096(v=vs.110).aspx).",
  "Link": "https://msdn.microsoft.com/en-us/library/hh367887(v=vs.110).aspx#winForms",
  "Markdown": null,
  "ApplicableApis": [
    "M:System.Drawing.Imaging.EncoderParameter.#ctor(System.Drawing.Imaging.Encoder,System.Int32,System.Int32,System.Int32,System.Int32)"
  ],
  "Related": null,
  "VersionBroken": "4.5",
  "VersionFixed": null,
  "IsBuildTime": true,
  "IsQuirked": false,
  "IsSourceAnalyzerAvailable": false,
  "BugLink": null,
  "Notes": null,
  "IsRetargeting": true,
  "ImpactScope": 2
}

**Parsed APIs:**
- [x] M:System.Drawing.Imaging.EncoderParameter.#ctor(System.Drawing.Imaging.Encoder,System.Int32,System.Int32,System.Int32,System.Int32)